samples/spring-security-hybrid-usage/k8s/deployment.yml

```diff
@@ -0,0 +1,214 @@
+---
+###  SERVICE INSTANCE CONFIGURATION  ###
+# Xsuaa instance
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceInstance
+metadata:
+  name: xsuaa-authn
+  namespace: <YOUR NAMESPACE>
+spec:
+  clusterServiceClassExternalName: xsuaa
+  clusterServicePlanExternalName: application
+  parameters:
+    xsappname: spring-security-hybrid-usage
+    tenant-mode: dedicated
+    scopes:
+      - name: "$XSAPPNAME.Read"
+        description: "Read Permissions."
+    role-templates:
+      - name: Viewer
+        description: View Data
+        scope-references:
+          - "$XSAPPNAME.Read"
+    role-collections:
+      - name: XSUAA-Viewer
+        description: Viewer (read)
+        role-template-references:
+          - "$XSAPPNAME.Viewer"
+---
+apiVersion: servicecatalog.k8s.io/v1beta1
```

Comment:
according to Jakob there is a successor of this service catalog, named BTP service operator:
https://github.com/SAP/sap-btp-service-operator/blob/main/README.md

Even this is still in alpha state (release planned for Q4) I think we should think about supporting only this but not catalog. With this plugin not each property is stored in its own file, but there is one file with all properties, such as clientid, certificate,...